contracts/token/ERC20/extensions/ERC20FlashMint.sol

```diff
@@ -43,6 +43,19 @@ abstract contract ERC20FlashMint is ERC20, IERC3156FlashLender {
         return 0;
     }
 
+    /**
+     * @dev Returns the reciever address of the flash fee. By default this
+     * implementation returns the address(0) which means the fee amount will be burnt.
+     * This function can be overloaded to change the fee reciever.
+     * @param token The token to be flash loaned.
+     * @return The address for which the flash fee will be sent to.
+     */
+    function _flashFeeReceiver(address token) internal view virtual returns (address) {
```

Comment:
I realized we don't need the `address token` argument, because the flash loaned token is always the token itself (`this`).